io.github.openhv.OpenHV.yaml

```diff
@@ -0,0 +1,63 @@
+app-id: io.github.openhv.OpenHV
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: openhv
+rename-desktop-file: openhv.desktop
+rename-appdata-file: openhv.metainfo.xml
+rename-icon: openhv
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - shared-modules/lua5.1/lua-5.1.5.json
+  - name: OpenHV
+    sources:
+      - dotnet-nuget.json
+      - openra-nuget.json
+      - openhv-nuget.json
+      - type: git
+        url: https://github.com/OpenRA/OpenRA
+        commit: e685731b337d76ba4d03b7dc84105029870236d4
+        dest: engine
+      - type: shell
+        commands:
+          - cd engine && make version VERSION=e685731b337d76ba4d03b7dc84105029870236d4
+      - type: file
+        url: https://github.com/OpenRA/GeoIP-Database/releases/download/monthly/IP2LOCATION-LITE-DB1.IPV6.BIN.ZIP
+        dest: engine
+        sha256: fe0c93e3c7ade60f4ffcf05b306ae9970544bc97065434af08dcf31b9d8383f4
+      - type: git
+        url: https://github.com/OpenHV/OpenHV
+        tag: 20220526
+        commit: 540deb65633ba85ab68878328fa18d0d5926191b
+      - type: patch
+        path: install.patch
+      - type: patch
+        path: man.patch
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/dotnet6/bin
+      append-ld-library-path: /usr/lib/sdk/dotnet6/lib
+      arch:
+        x86_64:
+          env:
+            RUNTIME: linux-x64
+        aarch64:
+          env:
+            RUNTIME: linux-arm64
+        arm:
+          env:
+            RUNTIME: linux-arm
```

Comment:
```suggestion
```

you don't need this,
(take 2)

Reply:
Okay. I only removed the binaries. By the way this was copied from https://github.com/flathub/org.freedesktop.Sdk.Extension.dotnet6